docs/release-strategy.md

```diff
@@ -0,0 +1,69 @@
+# InstructLab CLI Release Strategy
+
+This document discusses the release strategy and processes for the
+`instructlab` Python package built from the
+<https://github.com/instructlab/instructlab> git repository.
+
+## Versioning Scheme
+
+Releases use a `X.Y.Z` numbering scheme.
+
+X-stream release are for major releases. At this stage in the project a major release has not been cut and we expect each release (2-3 weeks) to be a new Y-stream.
+
+Backwards compatibility will exist within a Y-stream for example within 0.1.8 and 0.1.9 and between Y-stream releases for update purposes. For example between 0.1.9 and 0.2.0.
+
+Z-stream releases are meant for critical bug and documentation fixes. Backward compatibility is guaranteed between Z-stream releases. Z-stream releases are cut as maintainers see fit.
+
+## Schedule
+
+The project currently operates on a time-based release schedule.
+Before releasing version 1.0, the goal is to release a new Y-stream every 2-3 weeks though the time span remains flexible based on the discretion of the maintainers team.
+
+The cadence for major releases starting from 1.0 and onward will be determined as the project matures.
+
+A schedule will be updated in a markdown file on the <https://github.com/instructlab/instructlab> Github repository.
+
+## Release Tracking
+
+Release planning is tracked via milestones in Github. Milestones on Github will exist for the next two releases.
+
+Issues and PRs for milestones are set, monitored, and changed at weekly grooming sessions.
+Only PRs and issues associated with the next two milestones will be prioritized and merged into the main branch of instructlab
```

Comment:
"Only" seems too strong here. If as a maintainer, I jump on the PR queue and see a good bug fix that was just posted, merging it should be fine, even if it hasn't gone through a grooming session. It just seems like overhead that shouldn't be strictly always required.

```suggestion 
PRs and issues associated with the next two milestones will be prioritized for review and merging into the main branch of instructlab.
```